### PR TITLE
Remove ValuesPage, CulturePage, JobIndexPage and the impact_reports and sign_up_form apps

### PR DESCRIPTION
**Pull request by engineervix**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1349385054)

### Description of Changes Made

This PR removes the ValuesPage, CulturePage and JobIndexPage models from the codebase, as they are not needed. In addition, the `impact_reports app` app and the `sign_up_form` apps are removed, and may be added if needed in the future.

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [ ] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
- [x] Tests and linting passes.
- [ ] Consider updating documentation. If you don't, tell us why.
- [ ] If relevant, list the environments / browsers in which you tested your changes.

### CODE REVIEW

File: tbx/core/models.py [hunk 1]
```diff
@@ -17,7 +16,6 @@
 from wagtail.models import Orderable, Page
 from wagtail.snippets.models import register_snippet
 
-from .api import PeopleHRFeed
```
Comment (helenb) [inline tbx/core/models.py:20]: we could potentially remove the PeopleHRFeed code in the api as well.

File: tbx/people/models.py [hunk 2]
```diff
@@ -7,13 +7,11 @@
 from modelcluster.fields import ParentalKey
 from modelcluster.models import ClusterableModel
 from phonenumber_field.modelfields import PhoneNumberField
-from tbx.blog.models import BlogIndexPage, BlogPage
+from tbx.blog.models import BlogPage
 from tbx.core.blocks import PageSectionStoryBlock
 from tbx.core.utils.models import SocialFields
-from tbx.people.blocks import InstagramEmbedBlock, StandoutItemsBlock
```
Comment (helenb) [inline tbx/people/models.py:13]: the StandoutItemsBlock and InstagramEmbedBlock could now be removed along with any related templates if they are not used elsewhere

**Review by helenb — COMMENTED** [review]
Thanks @engineervix just a couple of comments for extra items that could be removed.

File: tbx/people/models.py (hunk 2 above)
Comment (engineervix) [reply tbx/people/models.py:13]: Thanks, removed in f925b0a and 95de04c

File: tbx/core/models.py (hunk 1 above)
Comment (engineervix) [reply tbx/core/models.py:20]: Thanks, removed in 5c0e485

**Comment by helenb** [comment]
I added a couple of comments directly on the commits but I don't think they are appearing - basically there may be some templates to remove, but I'll do a follow-up merge request.

I think we want to merge this straight to `main` not to the integration branch, as we need this for the data import

**Comment by helenb** [comment]
I'd like to do one more test of this before we merge it on my local build, but need to get some lunch now.

**Comment by helenb** [comment]
I've done a quick test and all looks good, thanks @engineervix

**Review by helenb — APPROVED** [review]

**Comment by engineervix** [comment]
> @engineervix there is another app we might want to remove that I think we discussed before christmas and that is the sign-up form page - I am not sure what it is used for. The models seem to be empty.

Indeed, I saw this while working on the migration and realised I forgot to remove it. Thank you, removing it now ...

**Pull request merged**